### PR TITLE
Release 3.3.2

### PR DESCRIPTION
**Pull request by Aryex**
### Summary

Patch release 3.3.2, include fixes for #476 

### Additional Reviewers

@alexey-temnikov 
@jeremyp-bq 
@alexr-bq 
@jonathanl-bq

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +53,11 @@ The methods for creating a spark cluster are documented [here](https://spark.apa
 
 Once you have a spark cluster, you can run such an application with spark-submit, including the connector jar.
 
+#### Spark compatibility
+
+The connector supports all Spark 3 releases. Our release numbers is of the major.minor.patch format, with the major 
+and minor numbers always matches that of the latest Spark 3 release.
+
```
Comment (jeremyp-bq) [inline README.md:60]: I don't think we need this as there is a section just above that states which versions we support and also in CONTRIBUTING.md.  Also, this seems to break up the sentence just above with the code snippet below.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,9 @@
 [![contributing](https://img.shields.io/badge/contributing-read-orange)](CONTRIBUTING.md)
 [![license](https://img.shields.io/badge/license-Apache%202.0-orange.svg)](https://opensource.org/licenses/Apache-2.0)
 
+![vertica-tested](https://img.shields.io/badge/Vertica%20Tested-10%20%7C%2011%7C%2012-blue)
+![spark-tested](https://img.shields.io/badge/Spark%20Tested-%3E%3D%203.0.0-blue)
```
Comment (jeremyp-bq) [inline README.md:14]: Need to be consistent with the versions and spacing (missing a space after the 11 in the Vertica versions).  I think the "|"-separated list of versions looks nicer and easier for customers to spot the version they use, but if we do use the ">=" instead we should do it for both Vertica and Spark versions.

File: README.md (hunk 1 above)
Comment (Aryex) [reply README.md]: Removed. Versioning information is now only in contributing.

File: README.md (hunk 2 above)
Comment (Aryex) [reply README.md]: updated to list out spark 3.0 3.1 etc

**Review by jeremyp-bq — APPROVED** [review]

**Pull request merged**